lib/validataion/jsonSchemas.js

```diff
@@ -201,6 +206,35 @@ schemas[validationKeys.ELEMENT_PROPS] = {
 				'val': {'type': 'boolean'},
 				deviation,
 			}},
+			{'properties': {
+				'name': {'const': ELEMENT_PROP.BORDER_STYLE},
+				'type': propComarator,
```

Comment:
'val' missing ?

Reply:
We have only apple border style values, maybe it has sense to add all possible border-style values so it will be possible to validate it with 'enum'

Reply:
I agree. @AntonSBilan, could you please add it in scope of this PR?

Reply:
Not only for border-style, but for other enumerable properties as well. Also, please update our user docs and Confluence specs to include this new constants to the list.